generator/ServiceClientGeneratorLib/Generators/Marshallers/JsonRPCResponseUnmarshaller.tt

```diff
@@ -132,7 +132,20 @@ namespace <#=this.Config.Namespace #>.Model.Internal.MarshallTransformations
             var responseBodyBytes = context.GetResponseBodyBytes();
 
             using (var streamCopy = new MemoryStream(responseBodyBytes))
+<#
+            if  (this.Config.ServiceModel.IsAwsQueryCompatible)
+            {
+#>  
+            using (var contextCopy = new JsonUnmarshallerContext(streamCopy, true, context.ResponseData))
```

Comment:
It's not obvious to me at a glance why we need to call this constructor differently - can you either explain here, or even better add a comment in the generator?